### PR TITLE
Sc buf stft 1024 chans

### PR DESCRIPTION
**Pull request by tedmoore**
Fixes (only by adding documentation for) https://github.com/flucoma/flucoma-sc/issues/22

This branch is branched from https://github.com/flucoma/flucoma-docs/pull/41, which should be pulled first.

### CODE REVIEW

**Review by weefuzzy — CHANGES_REQUESTED** [review]
Because there's a bit of noise in the PR from unrelated files with line-ending changes sneaking into some commit, it's a wee bit hard to follow this. AFAICT, you've introduced a new YAML field `scdescription` for appending SC special stuff to an object description block in the base template? 

I think if we want text in a specific reStrcuturedText scope to be filterable by environment, it might be preferable to find a rst way of approaching it if that's the way we're going anyway. It would be much more flexible than having to add new keys for each special case. 

re: the noisiness, two requests: 
* only commit the things you deliberately changed, and preferably review the changes before committing to make a minimal set pertinent to the PR
* stop your text editor changing line breaks on save (it always generates git noise) 

If your git-fu extends to reversing the commits to files with non-substantive changes (each driver file and the render.py?), then all the noise will happily go away when we squash-merge :heart:

**Comment by weefuzzy** [comment]
Maybe the [`container` directive in reStructuredText](https://docutils.sourceforge.io/docs/ref/rst/directives.html#container) can do what we want if we hack the python a wee bit 

```reStructuredText 

blah blah 

..container sc 
    more blah only in sc

..container max
   more blah only in max

```

**Comment by tedmoore** [comment]
> * only commit the things you deliberately changed, and preferably review the changes before committing to make a minimal set pertinent to the PR
> * stop your text editor changing line breaks on save (it always generates git noise)
> 
> If your git-fu extends to reversing the commits to files with non-substantive changes (each driver file and the render.py?), then all the noise will happily go away when we squash-merge ❤️

Yeah, I saw this and though, ah crap, that's really ugly. I have since investigated how to not make it look like that!

> If your git-fu extends to reversing the commits to files with non-substantive changes (each driver file and the render.py?), then all the noise will happily go away when we squash-merge ❤️

I'll see if I can--also in GH's diff there's an option to "ignore white space" which actually cleans it up quite well.

Screenshot 2022-01-28 at 19 59 23

> you've introduced a new YAML field scdescription for appending SC special stuff to an object description block in the base template?
 
correct.

> Maybe the container directive in reStructuredText can do what we want if we hack the python a wee bit

Cool. I will investigate.

**Comment by tedmoore** [comment]
@weefuzzy sorry I'm GH bombing you right now.

Is this `container` syntax supposed to go into the YAML? That seems a bit off to me? 

Any idea on the timeline for converting the YAML to rst? Should I go for the YAML implementation for now and then convert to rst containers when the rst conversion happens?

The extra field I'm adding to the YAML is completely invisible to the other CCE docs frameworks, so I don't think it'll hurt. (I would make another, clean, PR though).

**Comment by weefuzzy** [comment]
We already use rst in the YAML (which is nuts, I know, and why I'm proposing changing). My suggestion would need some python as well, in order to actually do something useful with the tag. I don't have a timeline for the changeover yet, because I'm stuck with something else. I guess I can live with one spurious block that needs clearing up as part of the conversion, if you're under pressure to get the SC boilerplate in place. But please only one 😀
